### ISSUE TITLE
Save as File

### ISSUE THREAD

**Issue by smaragdus** [LonMcGregor/TabExporter#2]
Hello,

**Tab Exporter** would be very useful for me if it could save the list of tabs as HTML file. Now hitting **Download as HTML** button just opens the list of tabs in HTML:

`blob:chrome-extension://nfplecfiodfcjnkkflgcgijnhhgglbgf/d4c41ff9-5ed9-43ba-bd51-10d967aaf448`

but I don't find a way to save this list as HTML file. Am I missing something?

For me the name of **Download as HTML** button is misleading- it does not download (save) the list of tabs as HTML, it just opens it. Perhaps current **Download as HTML** button should be renamed to **Open as HTML** and a new button- **Download as HTML** should be added which actually downloads (saves) the list of tabs as HTML file.

In short- **Tab Exporter** may have two buttons:

- **Open as HTML** button - it preserves the current behaviour of **Download as HTML** button;
- **Download as HTML** button - it downloads (saves) the list of tabs as HTML file;

Regards

**Comment by knivey** [comment]
Agreed the way this function now makes it useless, the page it opens if you try to save as a file you get a error, and if you try to view page source its blank. seems currently there is no way to actually save anything.
